q2_feature_table/plugin_setup.py

```diff
@@ -300,6 +300,41 @@
                 "on https://docs.qiime2.org for additional details."
 )
 
+plugin.methods.register_function(
+    function=q2_feature_table.filter_seqs,
+    inputs={
+        'data': FeatureData[Sequence],
```

Comment:
I'm not in love with `data`, but we use this in a [few](https://github.com/qiime2/q2-feature-table/blob/master/q2_feature_table/plugin_setup.py#L158-L159) places in [this](https://github.com/qiime2/q2-feature-table/blob/master/q2_feature_table/plugin_setup.py#L316) plugin.

Reply:
I'm not in love either -- what name would you choose if there wasn't a precedent for `data`?

Reply:
I like `seqs`, but then it becomes a bit stuttery with the action name...

Maybe `data` isn't the worst thing ever.

Reply:
Makes sense, let's just keep `data` (relates to [my comment](https://github.com/qiime2/q2-feature-table/pull/140#discussion_r138739903) above).